### PR TITLE
EWS send email integration

### PR DESCRIPTION
**Pull request by erezh31**

### CODE REVIEW

File: Integrations/integration-EWSMailSender.yml [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        use both.
+    - name: attachIDs
+      description: 'A comma-separated list of IDs of war room entries that contain
+        files. Used to attach files to the outgoing email. Example: attachIDs=15@8,19@8.'
+    - name: attachNames
+      description: A comma-separated list of names to the attachments to be send.
+        Should be the same number of elements as attachIDs.
+    description: Send email via EWS
+  dockerimage: demisto/py-ews:latest
+  runonce: false
```
Comment (meirwah) [inline Integrations/integration-EWSMailSender.yml]: no need for this key

File: Integrations/integration-EWSMailSender.yml [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      description: Send an html formatted email - overrides "body" argument, cannot
+        use both.
+    - name: attachIDs
+      description: 'A comma-separated list of IDs of war room entries that contain
+        files. Used to attach files to the outgoing email. Example: attachIDs=15@8,19@8.'
+    - name: attachNames
+      description: A comma-separated list of names to the attachments to be send.
+        Should be the same number of elements as attachIDs.
+    description: Send email via EWS
+  dockerimage: demisto/py-ews:latest
```
Comment (meirwah) [inline Integrations/integration-EWSMailSender.yml]: drop the :latest

File: Integrations/integration-EWSMailSender.yml [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+            'subject': subject,
+            'attachments': attachments_names
+        }
+
+        return {
+            'Type': entryTypes['note'],
+            'Contents': result_object,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown('Sent email', result_object),
```
Comment (meirwah) [inline Integrations/integration-EWSMailSender.yml]: no context fields?

File: Integrations/integration-EWSMailSender.yml [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+    config = prepare()
+    args = prepare_args(demisto.args())
+
+    if demisto.command() == 'test-module':
+        test_module()
+    elif demisto.command() == 'send-mail':
+        demisto.results(send_email(**args))
+  type: python
+  commands:
+  - name: send-mail
```
Comment (meirwah) [inline Integrations/integration-EWSMailSender.yml]: no outputs for this cmd?

Comment (erezh31) [reply Integrations/integration-EWSMailSender.yml]: no

File: Integrations/integration-EWSMailSender.yml (hunk 3 above)
Comment (erezh31) [reply Integrations/integration-EWSMailSender.yml]: no

**Comment by erezh31** [comment]
@meirwah  plz approve and merge

**Pull request merged**